code/scripting/api/libs/mission.cpp

```diff
@@ -99,7 +99,35 @@ ADE_FUNC(evaluateSEXP, l_Mission, "string", "Runs the defined SEXP script", "boo
 		return ADE_RETURN_FALSE;
 }
 
-ADE_FUNC(runSEXP, l_Mission, "string", "Runs the defined SEXP script", "boolean", "if the operation was successful")
+ADE_FUNC(evalSEXP, l_Mission, "string", "Runs the defined SEXP script, and returns the result as a boolean", "boolean", "true if the SEXP returned SEXP_TRUE; false if the SEXP returned anything else (even a number)")
```

Comment:
This seems to be an exact copy of the function above. Is that intentional?

Reply:
Yes.  I realized that in most cases the `( when ( true ) [sexp string] )` format isn't necessary, so `evaluateSEXP` is perfectly fine.  But I also wanted a shorter version of the name.

Reply:
I think it is not a good idea to add the same function twice just with a different name (unless there is deprecation involved). However, if you dislike the name you can also just rename it locally:
```lua
local evalSexp = mn.evaluateSEXP

evalSexp("( when ( true ) ( do-nothing ) )")
```

Reply:
Meh, ok